left-hand-nav-drill/main.css

```diff
@@ -1,7 +1,28 @@
 .container {
   margin: 0 auto;
-  /*Put your code here*/
```

Comment:
Your container is currently full width, you can set a `width` and/or `max-width` and it should center the container.